.github/workflows/release.yml

```diff
@@ -11,19 +11,19 @@ jobs:
 
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
     - name: Use Node.js
-      uses: actions/setup-node@v2
+      uses: actions/setup-node@v3
       with:
         node-version-file: '.nvmrc'
     - name: Install dependencies
-      run: npx ci
+      run: npx pnpm i --frozen-lockfile
     - name: Build
-      run: npm run build
+      run: npx pnpm build
     - name: Lint
-      run: npm run lint
+      run: npx pnpm lint
     - name: Test
-      run: npm run test --if-present
+      run: npx pnpm test --if-present
```

Comment:
Why are these changes necessary for pnp?

Reply:
Let's revert.